common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -513,13 +524,34 @@ def get_video_transcript_content(course_id, language_code, edx_video_id, youtube
         A dict containing transcript's file name and its sjson content.
     """
     transcript = None
-    if VideoTranscriptEnabledFlag.feature_enabled(course_id=course_id) and edxval_api:
+    if edxval_api:
         __, video_candidate_ids = get_video_ids_info(edx_video_id, youtube_id_1_0, html5_sources)
         transcript = edxval_api.get_video_transcript_data(video_candidate_ids, language_code)
 
     return transcript
 
 
+def get_available_transcript_languages(edx_video_id, youtube_id_1_0, html5_sources):
+    """
+    Gets available transcript languages from edx-val, only if the corresponding
+    feature flag is enabled for the given `course_id`.
```

Comment:
I think this part `only if the corresponding feature flag is enabled for the given `course_id`` Since we're not checking in this method.